src/Microsoft.PowerShell.Commands.Management/commands/management/GetClipboardCommand.cs

```diff
@@ -35,13 +38,40 @@ public enum ClipboardFormat
         Audio = 3,
     };
 
+#if UNIX
+    /// <summary>
+    /// Defines the different text formats supported by clipboard.
+    /// </summary>
+    public enum TextDataFormat
+    {
+        /// Text format as default.
```

Comment:
We need "summary" tag.
Below too.

Reply:
Will add although CodeFactor didn't complain...